ghost/admin/app/services/migrate.js

```diff
@@ -63,6 +64,19 @@ export default class MigrateService extends Service {
         });
     }
 
+    async ownerUser() {
```

Comment:
You have the correct pattern here, but this function already exists in the billing service `getOwnerUser()`. 

We can add `@service billing` to line 10 of `gh-migrate-iframe` and use that.

Or, you could go one better and move that function out of the billing service, into a new "owner" service that's used in both places - which could be more easily tested.